app/assets/javascripts/semantic-ui.js

```diff
@@ -0,0 +1,19 @@
+//= require semantic-ui/modules/accordion.js 
+//= require semantic-ui/modules/behavior/api.js 
+//= require semantic-ui/modules/behavior/colorize.js 
+//= require semantic-ui/modules/behavior/form.js 
+//= require semantic-ui/modules/behavior/state.js 
+//= require semantic-ui/modules/chatroom.js 
+//= require semantic-ui/modules/checkbox.js 
+//= require semantic-ui/modules/dimmer.js 
+//= require semantic-ui/modules/dropdown.js 
+//= require semantic-ui/modules/modal.js 
+//= require semantic-ui/modules/nag.js 
+//= require semantic-ui/modules/popup.js 
+//= require semantic-ui/modules/rating.js 
+//= require semantic-ui/modules/search.js 
+//= require semantic-ui/modules/shape.js 
+//= require semantic-ui/modules/sidebar.js 
+//= require semantic-ui/modules/tab.js 
+//= require semantic-ui/modules/transition.js 
+//= require semantic-ui/modules/video.js 
```

Comment:
We avoid to mix and match different styling frameworks

Reply:
How do you mean?

Reply:
ah, now i understand.
Note taken!